pkg/kubelet/cm/node_container_manager_linux_test.go

```diff
@@ -400,3 +401,51 @@ func getEphemeralStorageResourceList(storage string) v1.ResourceList {
 	}
 	return res
 }
+
+func TestUpdateNodeAllocatableCgroups(t *testing.T) {
```

Comment:
nit: after the rewrite, testing this logic by testing `updateInternalCapacity` makes more sense, as you then don't have to think about the errors from cgroup enforcement.

Reply:
I've added a test for `updateInternalCapacity()` and extended it to catch incomplete updates. But left `TestUpdateNodeAllocatableCgroups` mostly untouched. Probably it could be degenerated to a simple call to `UpdateNodeAllocatableCgroups()`. idk